mito-ds/monorepo
# Turn on code config for mitosheet private

# Description

Turn on code config for mitosheet private, without using the mito_config because this isn't a publicly configurable thing. We'd rather handle this ourselves. 

# Testing

- Make sure the code config is not turned on when on pro
- Make sure the code config is turned on when using enterprise
- Make sure the code config is turned on when you set the package_name to 'mitosheet-private'

# Documentation

Note if any new documentation needs to addressed or reviewed.
